.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature Request
```

Comment:
Should we also ask the value the request will provide (either as its own field or (more likely) in the comment below the "brief summary" section?